src/shared/ui/components/BaseTable.vue

```diff
@@ -115,8 +114,8 @@ const { width } = useWindowSize();
 const PAGINATION_BREAKPOINT = 960;
 
 const items = computed(() => {
-  if (props.loading && props.pagination) {
-    return Array.from({ length: props.pagination.page_size }, () => null);
+  if (props.loading) {
+    return Array.from({ length: props.pagination?.page_size ?? 10 }, () => null);
   }
```

Comment:
I don't immediately like this assumption of 10 by default. Can we leave it as it was for now?

Reply:
Fixed✅